client/web/src/enterprise/campaigns/list/CampaignsListEmpty.tsx

```diff
@@ -0,0 +1,214 @@
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div className="col-12 col-md-6 mb-2">
+                <div className="card h-100 p-2">
+                    <div className="card-body d-flex">
+                        <RefactorCombyIcon className="mr-3" />
+                        <div>
+                            <h4>
+                                <a
+                                    href="https://docs.sourcegraph.com/campaigns/tutorials/updating_go_import_statements"
+                                    rel="noopener"
+                                >
+                                    Refactoring with language aware search
+                                </a>
+                            </h4>
+                            <p className="text-muted mb-0">
+                                Using comby language aware search to refactor Go statements to a semantically
+                                equivalent, but clearer execution.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <p>
+            <a href="https://docs.sourcegraph.com/campaigns/tutorials" rel="noopener">
+                More tutorials
+            </a>
+        </p>
+        <div className="row mb-4">
+            <div className="col-12 col-md-4 mb-3">
+                <p>
+                    <strong>Quickstart</strong>
+                </p>
+                <p>Create your first Sourcegraph campaign in 10 minutes or less by altering a readme.</p>
+                <a href="https://docs.sourcegraph.com/campaigns/quickstart" rel="noopener">
+                    Campaigns quickstart
+                </a>
+            </div>
+            <div className="col-12 col-md-4 mb-3">
+                <p>
+                    <strong>Introduction</strong>
+                </p>
+                <p>Learn how campaigns enables large-scale code changes across many repositories and code hosts.</p>
+                <a href="https://docs.sourcegraph.com/campaigns/explanations/introduction_to_campaigns" rel="noopener">
+                    Campaigns introduction
+                </a>
+            </div>
+            <div className="col-12 col-md-4 mb-3">
+                <p>
+                    <strong>Documentation</strong>
+                </p>
+                <p>
+                    Learn about the campaign spec{' '}
+                    <a
+                        href="https://docs.sourcegraph.com/campaigns/references/campaign_spec_yaml_reference"
+                        rel="noopener"
+                    >
+                        YAML reference
+                    </a>
+                    , it's powerful{' '}
+                    <a href="https://docs.sourcegraph.com/campaigns/references/campaign_spec_templating">
+                        templating language
+                    </a>
```

Comment:
You don't learn _about_ a campaign spec YAML reference, you look at it or use it to learn something.

```suggestion
                    Take a look at the{' '}
                    <a
                        href="https://docs.sourcegraph.com/campaigns/references/campaign_spec_yaml_reference"
                        rel="noopener"
                    >
                       campaign spec YAML reference
                    </a>
                    , learn about its powerful{' '}
                    <a href="https://docs.sourcegraph.com/campaigns/references/campaign_spec_templating">
                        templating language
                    </a>
```